evap/staff/views.py

```diff
@@ -1014,21 +1028,28 @@ def evaluation_person_management(request, semester_id, evaluation_id):
                 if not errors:
                     save_import_file(excel_file, request.user.id, import_type)
 
-        elif 'import' in operation:
-            file_content = get_import_file_content_or_raise(request.user.id, import_type)
-            success_messages, warnings, __ = PersonImporter.process_file_content(import_type, evaluation, test_run=False, file_content=file_content)
-            delete_import_file(request.user.id, import_type)
+        else:
+            additional_messages = []
+            if 'replace' in operation:
+                deleted_person_count, deletion_message = helper_delete_users_from_evaluation(evaluation, operation)
+                additional_messages = format_html(deletion_message, deleted_person_count, evaluation.name)
```

Comment:
please use `evaluation.full_name` instead of `evaluation.name`. the `name` can be empty, `full_name` would then use the course's name instead.